dts/arm/nxp/nxp_imx6sx_m4.dtsi

```diff
@@ -241,6 +241,18 @@
 			#gpio-cells = <2>;
 			status = "disabled";
 		};
+
+		mub:mu@4229C000 {
```

Comment:
Can you make this 4229C000 -> 4229c000 (and the reg as well)

Reply:
Done.